### PR TITLE
🌱 openapi-gen use upstream binary

### PR DESCRIPTION
**Pull request by chrischdi**
Also changes a comment in the generated files so we don't have to change the boilerplate generation anymore.

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: hack/boilerplate/boilerplate.py [hunk 1]
```diff
@@ -215,7 +215,7 @@ def get_regexs():
     # strip #!.* from scripts
     regexs["shebang"] = re.compile(r"^(#!.*\n)\n*", re.MULTILINE)
     # Search for generated files
-    regexs["generated"] = re.compile('DO NOT EDIT|Do not edit it manually!')
+    regexs["generated"] = re.compile('DO NOT EDIT')
```
Comment (sbueringer) [inline hack/boilerplate/boilerplate.py:218]: is that rollback intended?

Comment (sbueringer) [reply hack/boilerplate/boilerplate.py:218]: I assume we're not using the python script anymore?

File: hack/tools/runtime-openapi-gen/main.go [hunk 2]
```diff
@@ -59,19 +53,25 @@ func main() {
 	}
 
 	c := catalog.New()
-	_ = runtimehooksv1.AddToCatalog(c)
-	_ = v1alpha2.AddToCatalog(c)
-	_ = v1alpha3.AddToCatalog(c)
+	if err := runtimehooksv1.AddToCatalog(c); err != nil {
```
Comment (sbueringer) [inline hack/tools/runtime-openapi-gen/main.go:56]: You can ignore the errors as before like we always do when we add a package to a scheme. We followed the pattern from scheme and are just panicking if something goes wrong

Comment (sbueringer) [reply hack/tools/runtime-openapi-gen/main.go:56]: Actually the only error which can occur is actually from the upstream scheme, so I would ignore it just like we do usually with that one

**Review by sbueringer — COMMENTED** [review]
just a few questions/nits

File: hack/boilerplate/boilerplate.py (hunk 1 above)
Comment (chrischdi) [reply hack/boilerplate/boilerplate.py:218]: the upstream openapi-gen adds a `DO NOT EDIT` so the `|Do not edit it manually!` (which I added 1 or two commits ago) is reverted.

Comment (sbueringer) [reply hack/boilerplate/boilerplate.py:218]: Fun, so that came from us somehow? :)

Comment (chrischdi) [reply hack/boilerplate/boilerplate.py:218]: came from our custom main.go

**Pull request merged**